src/test/integration/IntegrationDeployer.t.sol

```diff
@@ -699,59 +649,62 @@ abstract contract IntegrationDeployer is ExistingDeploymentParser {
 
     /**
      * @dev Create a new User with a random config using the range defined in `_configRand`
-     * 
+     *
      * Assets are pulled from `strategies` based on a random staker/operator `assetType`
      */
-    function _randUser(string memory name) internal returns (User, IStrategy[] memory, uint[] memory) {
+    function _randUser(
+        string memory name
+    ) internal returns (User, IStrategy[] memory, uint256[] memory) {
         // For the new user, select what type of assets they'll have and whether
         // they'll use `xWithSignature` methods.
         //
         // The values selected here are in the ranges configured via `_configRand`
-        uint assetType = _randAssetType();
-        uint userType = _randUserType();
-        
+        uint256 assetType = _randAssetType();
```

Comment:
Can we change uint256 to uint in a separate PR? We went with uint everywhere just to make it simple to read (and type). Also helps with review in terms of reviewing less

Reply:
fixed in [4574f6e](https://github.com/Layr-Labs/eigenlayer-contracts/pull/894/commits/4574f6e615518ffc784794c3be9e53ddbf2f90e6)